src/Amadeus/Client/RequestOptions/Fare/MPDate.php

```diff
@@ -101,4 +101,11 @@ class MPDate extends LoadParamsFromArray
      * @var int
      */
     public $range;
+
+    /**
+     * Details of the trip duration
```

Comment:
You should indicate in a PHPDOC remark that this param isn't supported yet in Amadeus